collections/harvestparams.php

```diff
@@ -294,7 +294,7 @@
 					<label for="hascoords"><?php echo isset($LANG['HAS_COORDS'])?$LANG['HAS_COORDS']:'Limit to Specimens with Geocoordinates Only'; ?></label>
 				</div>
 				<div>
-					<input type='checkbox' name='includecult' id='includecult' value='1' />
+					<input type='checkbox' name='includecult' id='includecult' value='<?php echo $CSS_VERSION_RELEASE === 'uswds' ? '1' : '0' ?>' <?php echo $CSS_VERSION_RELEASE === 'uswds' ? 'checked' : '' ?> />
```

Comment:
Happy to change this to a configuration in symbini.php that doesn't depend on the styling. Especially if we foresee this being something other portals besides the USDA would want.

Reply:
I went ahead and changed this in 4e09df7ed.